configs/config.json.map.example

```diff
@@ -455,22 +455,43 @@
 		"config": {
 			"enabled": true,
 			"mode": "social",
-			"bullets": 3,
+			"bullets": 1,
 			"homing_shots": true,
 			"special_iv": 100,
-			"order": ["missing", "iv", "priority", "vip", "expiration_timestamp_ms"],
+			"order": ["missing", "vip", "priority"],
 			"sources": [
 				{
 					"url": "http://localhost:5000/raw_data",
 					"key": "pokemons",
```

Comment:
Please add `"enabled": false`

I suggest disabling all sources by default, allowing user to enable exactly what he wants to